### ISSUE TITLE
Unable to parse date range from input value

### ISSUE THREAD

**Issue by hagen93** [Eonasdan/tempus-dominus#2871]
Hi,

I was trying to initialize the picker from the input fields HTML value attribute, but I'm just ending up with erroneous single dates instead of ranges. Seems to me that the error might be in this line: https://github.com/Eonasdan/tempus-dominus/blob/4a348cb3f7c60cc2db6908c3154da7189e148e92/src/js/tempus-dominus.ts#L534 which probably should do the parsing for multiple in cases where dateRange is true too, e.g. condition might be `this.optionsStore.options.multipleDates || this.optionsStore.options.dateRange`.

I tried enabling `multipleDates` as a workaround, but that seems to be invalid configuration when `dateRange` is enabled.

**Comment by hagen93** [comment]
For reference my configuration is as follows:
```
{
    dateRange: true,
    multipleDatesSeparator: ' - ',
    display: {
        components: {
            clock: false
        }
    },
    localization: {
        format: 'dd/MM/yyyy',
    }
}
```

**Comment by Eonasdan** [comment]
Hello. This was released with 6.7.19
